### PR TITLE
TS3 (creating UserInfo class) + TS7 (making a command to populate default members in the db) changes.

### PR DESCRIPTION
**Pull request by jleodones**

### CODE REVIEW

**Comment by jleodones** [comment]
[TS-3: Create a User model to store users](https://trello.com/c/xVGe82Ar/3-ts-3-create-a-user-model-to-store-users)

**Comment by jleodones** [comment]
[TS-7: Create management commands that populates basicclassinfo, basicclasssection, instructor](https://trello.com/c/dPZDGwXD/7-ts-7-create-management-commands-that-populates-basicclassinfo-basicclasssection-instructor)

File: apps/users/models.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+from apps.classes.models import BasicClassSection
+
+
 # Create your models here.
+class UserInfo(models.Model):
+    """
+    Inherits from the User class model, allowing authentication with the server.
+    """
+
+    user = models.OneToOneField(User)
```
Comment (markkyaw) [inline apps/users/models.py]: add `related_name="info"` and `on_delete=models.CASCADE`

File: apps/classes/models.py [hunk 2]
```diff
@@ -74,6 +73,7 @@ class BasicClassSection(BasicClassInfo):
     - Maybe implement default values for future semesters when not all values have been assigned.
     """
 
+    class_info = models.ForeignKey(BasicClassInfo)
```
Comment (markkyaw) [inline apps/classes/models.py]: add `related_name="class_section"` and `on_delete=models.CASCADE`

File: apps/classes/management/commands/populate_default.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+class Command(BaseCommand):
+    help = "Populates 5 BasicClassInfos, 5 BasicClassSessions, 5 Instructors, 5 Users"
+
+    def handle(self, *args, **kwargs):
+
+        # Populating UserInfo.
+        ui_list = ["user1", "user2", "user3", "user4", "user5"]
+
+        for user in ui_list:
+            User.objects.create_user(user, user + "@gmail.com", user + "pw")
```
Comment (markkyaw) [inline apps/classes/management/commands/populate_default.py]: Put this in a try catch block
```
try:
  # check if user exists by getting user object based on username/email
except:
  # create new user object
```

File: apps/classes/management/commands/populate_default.py [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        # Populating Instructors.
+        instructors_list = {
+            "Sanjay Madhav": "",
+            "David Matthias Kempe": "",
+            "Margaret Moser": "",
+            "Reuven Firestone": "",
+            "Aaron Cote": "",
+        }
+
+        for key, value in instructors_list:
```
Comment (markkyaw) [inline apps/classes/management/commands/populate_default.py]: iterate using `instructors_list.items()`

File: apps/classes/management/commands/populate_default.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            value = Instructor.objects.create(name=key)
```
Comment (markkyaw) [inline apps/classes/management/commands/populate_default.py]: i realized can't update dict like that lmfao
`instructors_list[key] = ...`

And do `update_or_create` <- so we don't create duplicate objects when we run this multiple times

File: apps/classes/management/commands/populate_default.py [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+        # Populating BasicClassInfos with Spring 2022 classes.
+        classinfo_list = {
+            "itp_380": BasicClassInfo.objects.create(
```
Comment (markkyaw) [inline apps/classes/management/commands/populate_default.py]: `update_or_create` for all of them in here

File: apps/classes/management/commands/populate_default.py [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+                year=2022,
+                semester=1,
+                min_units=4.0,
+                max_units=4.0,
+            ),
+        }
+
+        # Populating BasicClassSessions.
+        classsections_list = {
+            "itp380_lecture": BasicClassSection.objects.create(
```
Comment (markkyaw) [inline apps/classes/management/commands/populate_default.py]: `update_or_create`

**Review by markkyaw — CHANGES_REQUESTED** [review]
And try running 
`python3 manage.py populate_default` and try to debug the errors :D

File: apps/classes/management/commands/populate_default.py (hunk 5 above)
Comment (markkyaw) [reply apps/classes/management/commands/populate_default.py]: Doc: https://docs.djangoproject.com/en/4.0/ref/models/querysets/#update-or-create
It returns a tuple so look at what we need to save

**Review by markkyaw — DISMISSED** [review]

**Review by markkyaw — APPROVED** [review]

**Pull request merged**